### PR TITLE
Update readme for mean.js 0.4.x

### PR DESCRIPTION
**Pull request by boykoc**
Updating the readme.md to clarify what the yo generator currently does.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,9 @@ $ wget https://github.com/meanjs/mean/archive/master.zip -O meanjs.zip; unzip me
 Don't forget to rename **mean-master** after your project name.
 
 ### Yo Generator
--Another way would be to use the [Official Yo Generator](http://meanjs.org/generator.html), which generates a copy of the MEAN.JS 0.3.x boilerplate and supplies multiple sub-generators to ease your daily development cycles. 
+Another way would be to use the [Official Yo Generator](http://meanjs.org/generator.html), which generates a copy of the MEAN.JS 0.4.x boilerplate and supplies an application generator to ease your daily development cycles. 
+
+Currently there are no sub generators for the current version of the MEAN.js project, but they are under development and will be coming soon.
```
Comment (lirantal) [inline README.md]: Is the extra info about sub-generators really required?
I think we can avoid that.

**Comment by lirantal** [comment]
LGTM

**Comment by mleanos** [comment]
LGTM. It's good that this one is just in time to be released with 0.4.2 :) Thanks @boykoc

File: README.md (hunk 1 above)
Comment (mleanos) [inline README.md]: I see @lirantal point. That extra info might not be needed. It's sort of a moot point, in regards to it's relationship with this project. It seems that this type of statement is better suited for the generator Readme.

**Comment by boykoc** [comment]
@mleanos and @lirantal Happy to help in this small way.  Let me know if you'd like me to remove line 76. I can see how it is slightly off-topic plus line 74 now states there is only an app generator.

File: README.md (hunk 1 above)
Comment (ilanbiala) [inline README.md]: Unnecessary since it's in the generator README.

**Comment by codydaig** [comment]
Let's remove that line and update the commit message to meet the guidelines then LGTM. 

https://github.com/meanjs/mean/blob/master/CONTRIBUTING.md#commit-message-guidelines

**Comment by mleanos** [comment]
@boykoc Is it possible for you to squash to one commit? The new commit message looks good, but having the other commit will throw things off with the changelog.

**Comment by codydaig** [comment]
@boykoc Squash then  LGTM

**Comment by boykoc** [comment]
@codydaig squashed.  Sorry for all the back and forth for such a small update.

**Comment by codydaig** [comment]
@boykoc No worries! Always glad to have people contributing. We've just changed the commit message guidelines recently so we know that's going to be a pain point across the board. Welcome! :-D

**Pull request merged**